pages/linux/lrztar.md

```diff
@@ -0,0 +1,24 @@
+# lrztar
+
+> A wrapper for `lrzip` to simplify compression of directories.
+> See also: `lrzuntar`, `lrunzip`.
+
+- Archive a directory with tar, then compress:
+
+`lrztar {{path/to/directory}}`
+
+- Same as above, with ZPAQ - extreme compression, but very slow:
+
+`lrztar -z {{path/to/directory}}`
+
+- Specify the output file name and/or path:
+
+`lrztar -o {{path/to/output_file}} {{path/to/directory}}`
```

Comment:
path/to/output_file_or_folder according to https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md#token-syntax